src/react/components/pages/editorPage/canvasHelpers.ts

```diff
@@ -27,6 +27,89 @@ export default class CanvasHelpers {
         }
     }
 
+    /**
+     * Adds region to regions if missing,
+     * Removes region from regions if contained
+     */
+    public static toggleRegion(regions: IRegion[], region: IRegion): void {
+        const index = regions.findIndex((r) => r.id === region.id);
+        if (index === -1) {
+            regions.push(region);
+        } else {
+            regions.splice(index, 1);
+        }
+    }
+
+    /**
```

Comment:
Not sure these add much value (finde, findRegion, findIndex) since the replaced code is essentially the same... Thoughts?

Reply:
True. I was using it a lot more when I was refactoring, and thought I was saving some time. But I'll take these pieces out